src/components/HathorFlatList.component.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  FlatList, FlatListProps, StyleSheet, View,
```

Comment:
`FlatListProps` not being used as per the warning

Reply:
It was removed.